python/lsst/obs/base/ingest_tests.py

```diff
@@ -284,9 +305,32 @@ def testInPlace(self):
         """
         # symlink into repo root manually
         butler = Butler(self.root, run=self.outputRun)
-        pathInStore = "prefix-" + os.path.basename(self.file)
+
+        # If we have an index file in the source directory we need to
+        # create a symlink for that as well but we also have to reuse the
+        # test file name and not make a new name unless we change the
+        # content of the index
```

Comment:
"unless we change the content of the index" - I don't see how this applies to the test. If the contents of the index change, does the test have to change, or something else?

Reply:
The filename is hardcoded into the index file. That's all I'm saying.